specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2696 @@
+            "type": "string"
+          },
+          "supportedAggregationTypes": {
+            "description": "supported aggregation types",
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "supportedTimeGrainTypes": {
+            "description": "supported time grain types",
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "unit": {
+            "description": "units for the metric",
+            "type": "string"
+          }
+        }
+      },
+      "OperationMetaLogSpecification": {
+        "description": "log specifications for operation api",
+        "type": "object",
+        "properties": {
+          "blobDuration": {
+            "description": "blob duration of the log",
+            "type": "string"
+          },
+          "displayName": {
+            "description": "localized name of the log category",
+            "type": "string"
+          },
+          "name": {
+            "description": "name of the log category",
+            "type": "string"
+          }
+        }
+      },
+      "DimensionProperties": {
+        "description": "properties for dimension",
+        "type": "object",
+        "properties": {
+          "displayName": {
+            "description": "localized display name of the dimension to customer",
+            "type": "string"
+          },
+          "name": {
+            "description": "dimension name",
+            "type": "string"
+          },
+          "toBeExportedForCustomer": {
+            "description": "flag indicating whether this dimension should be included to the customer in Azure Monitor logs (aka Shoebox)",
+            "type": "boolean"
+          }
+        }
+      },
+      "PrivateEndpointConnectionList": {
+        "description": "Paged list of private endpoint connections",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "count": {
+            "format": "int64",
+            "description": "Total item count.",
+            "type": "integer"
+          },
+          "nextLink": {
+            "description": "The Url of next result page.",
+            "type": "string"
+          },
+          "value": {
+            "description": "Collection of items of type results.",
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/PrivateEndpointConnection"
+            }
+          }
+        }
+      },
+      "PrivateLinkResourceList": {
+        "description": "Paged list of private link resources",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "count": {
```

Comment:
List GET endpoints (collection GET) must only have value and nextLink in properties.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-collection-only-has-value-nextlink.md